### PR TITLE
Add helm 3.1.0 and helm-push addon

### PR DESCRIPTION
**Pull request by mrWinston**
This PR adds Helm and the helm-push addon to the builder image. 

To test, run the following: (@NandoTheessen )

```
docker build -f python.Dockerfile -t 3yourmind/jenkins-node-config:helm . 
docker run -it --rm --entrypoint "helm" 3yourmind/jenkins-node-config:helm version
```

### CODE REVIEW

**Comment by NandoTheessen** [comment]
Is that a PR label :o

File: python.Dockerfile [hunk 1]
```diff
@@ -2,6 +2,7 @@ FROM jenkins/jnlp-slave:alpine
 
 ENV SCANNER_VERSION 3.3.0.1492
 ENV DOCKER_VERSION=18.09.5
+ENV HELM_VERSION=3.1.0
```
Comment (jkhlr) [inline python.Dockerfile]: We are currently using helm version 3.0.3, I'm not sure if this is compatible...

**Review by jkhlr — CHANGES_REQUESTED** [review]

File: python.Dockerfile (hunk 1 above)
Comment (mrWinston) [reply python.Dockerfile]: Makes sense! I have been using 3.1.0 already, but we should aim to use our tried and tested version and update this once we're confident there aren't any problems.

**Review by NandoTheessen — APPROVED** [review]
lgtm!

**Review by jkhlr — APPROVED** [review]

**Pull request merged**